movingSquare.js

```diff
@@ -0,0 +1,127 @@
+const initMovingSquare = (() => {
+
+  const createSquare = (containerSelector) => {
+
+    const container = document.querySelector(containerSelector)
+
+    if (!container) return
+
+    const square = document.createElement('div')
+
+    square.style.backgroundColor = 'red'
+    square.style.width = '100px'
+    square.style.height = '100px'
+    square.style.position = 'absolute'
+    square.style.left = '0px'
+    square.style.top = '0px'
+
+    container.appendChild(square)
+
+    return square
+  }
+  const createParagraph = (containerSelector) => {
+
+    const container = document.querySelector(containerSelector)
+
+    if (!container) return
+
+    const p = document.createElement('p')
+
+    p.style.position = 'absolute'
+    p.style.width = '1px'
+    p.style.height = '1px'
+    p.style.left = '35px'
+    p.style.top = '17px'
+
+    container.appendChild(p)
+
+    return p
+  }
+
+  const square = createSquare('body')
+  const p = createParagraph('div')
+
+  let positionX = 0
```

Comment:
This is application state. I personally would put this on the top of the funciton.